### ISSUE TITLE
issue funguje

### ISSUE THREAD

**Issue by lietava** [mbombara/hello-world#2]
Je vlavo hore hned vedla <>Code

**Comment by mbombara** [comment]
Ano, cize tu u mna to funguje, a to som Ta len pozval. Stale vsak to iste nevidim u Teba (skusal som aj menit branch, no nepomohlo).. Bude patrat..
